routes/auth.routes.js

```diff
@@ -0,0 +1,194 @@
+const express = require('express');
+const router = express.Router();
+
+const User = require('../models/User.model');
+const Recipe = require('../models/Recipe.model');
+
+const bcrypt = require('bcrypt');
+const bcryptSalt = 10;
+const passport = require('passport');
+
+const nodemailer = require('nodemailer');
+
+// SMTP
+let transporter = nodemailer.createTransport({
+  service: 'Gmail',
+  auth: {
+    user: 'tester123.peterpan@gmail.com',
+    pass: '89675rutitgzrvuz',
```

Comment:
this you will want to move to the `.env` file and then use `process.env.EMAIL_USER` etc.